pokemongo_bot/cell_workers/move_to_map_pokemon.py

```diff
@@ -111,8 +111,8 @@ def pokemons_parser(self, pokemon_list):
         for pokemon in pokemon_list:
             disappear = int(pokemon.get('expiration_timestamp_ms', 0) / 1000) or int(pokemon.get('disappear_time', 0) / 1000)
 
-            pokemon['encounter_id'] = pokemon['encounter_id']
-            pokemon['spawn_point_id'] = pokemon['spawn_point_id']
+            pokemon['encounter_id'] = pokemon.get('encounter_id', '')
+            pokemon['spawn_point_id'] = pokemon.get('spawn_point_id', '') or pokemon.get('spawnpoint_id', '')
```

Comment:
I'm not sure on social though, but I'm planning to enable dynamic param name mapping.